e2e/etcd_test.go

```diff
@@ -175,8 +175,8 @@ func newEtcdProcessCluster(cfg *etcdProcessClusterConfig) (*etcdProcessCluster,
 		go func(etcdp *etcdProcess) {
 			_, err := etcdp.proc.ExpectRegex(readyStr)
 			readyC <- err
-			etcdp.proc.ReadUntil('\n') // don't display rest of line
-			etcdp.proc.Interact()
```

Comment:
That's a go idiom afaik-- the goroutines should shutdown on EOF from closing the process. The problem with blocking on stdout is if etcdserver tries to write to the log, it'll stall forever and won't make forward progress.